worldgen/tests/test_zone_overlap_policy.py

```
@@ -0,0 +1,319 @@
+    }
+    if dimension is not None:
+        zone["dimension"] = dimension
+    return zone
+
+
+class ZoneOverlapPolicyTest(unittest.TestCase):
+    def test_only_reviewed_zone_pairs_overlap_in_three_dimensions(self) -> None:
+        for path in FILES:
+            zones = _zones(path)
+            known_defects = KNOWN_DEFECT_OVERLAPS_BY_FILE[path.name]
+            actual, missing_designed, unexpected, stale_known = _policy_violations(
```

Ruff (0.15.21)

[warning] Unpacked variable `actual` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)